File: News_Service/src/main/java/com/dunice/projectNews/repository/TagRepository.java
package com.dunice.projectNews.repository;

import com.dunice.projectNews.models.Tag;
import org.springframework.data.jpa.repository.JpaRepository;
import java.util.Optional;

public interface TagRepository extends JpaRepository<Tag, Long> {

    Optional<Tag> findByTitle(String title);

}

File: News_Service_Kotlin/src/main/kotlin/com/dunice/advancedServiceKotlin/repository/TagRepository.kt
package com.dunice.advancedServiceKotlin.repository

import com.dunice.advancedServiceKotlin.models.Tag
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository
import java.util.*
@Repository
interface TagRepository : JpaRepository<Tag, Long> {
    fun findByTitle(title: String?): Optional<Tag>?
}
